src/eterna/pose2D/Pose2D.ts

```diff
@@ -2871,6 +2872,23 @@ export default class Pose2D extends ContainerObject implements Updatable {
         this._prevOffsetY = this._offY;
     }
 
+    public lateUpdate(_dt: number): void {
```

Comment:
Solid changes!

Reply:
Wish I didn't have to do the lateUpdate like this, but it does resolve the occasional bugged positioning!